src/frequenz/sdk/timeseries/_formula_engine/_tokenizer.py

```diff
@@ -79,16 +79,24 @@ class TokenType(Enum):
     """Represents the types of tokens the Tokenizer can return."""
 
     COMPONENT_METRIC = 0
+    """A component metric ID."""
+
     CONSTANT = 1
+    """A constant value."""
+
     OPER = 2
+    """An operator."""
 
 
 @dataclass
 class Token:
     """Represents a Token returned by the Tokenizer."""
 
     type: TokenType
+    """The type of the token."""
+
     value: str
+    """The value of the token."""
```

Comment:
Maybe "The value associated to the token"?

Reply:
```suggestion
    """The value associated to the token."""
```